percona-packaging/debian/rules

```diff
@@ -11,7 +11,7 @@ export TOOLS_TAGS=ssl sasl
 export PORTABLE=1
 export USE_SSE=1
 export PATH := /usr/local/go/bin:$(PATH)
-export FIPSMODE=0
+export FULL_FEATURED=1
```

Comment:
I don't see where this variable is used. If it is really unused, probably it's worth removing it completely.